### PR TITLE
Fix flaky tests

### PR DESCRIPTION
**Pull request by tiagolobocastro**
test(csi/parameters): use actual device path
    
    Test had mistakenly been using a hardcoded device path..
    
    Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

---

    test(events): increase timeout
    
    Should hopefully fix some random CI failures.
    
    Signed-off-by: Tiago Castro <tiagolobocastro@gmail.com>

### CODE REVIEW

**Comment by tiagolobocastro** [comment]
> file = f"/sys/block/nvme2c2n1/queue/io_timeout"

Not sure how that got checked in :D

**Review by dsavitskiy — APPROVED** [review]

File: tests/bdd/features/csi/node/test_parameters.py [hunk 1]
```diff
@@ -54,19 +54,18 @@ def staging_a_volume(staging_a_volume):
...
     print(f"queue_count: {queue_count}")
     # max io queues is cpu_count
     # admin q is 1
     assert queue_count == min(total, os.cpu_count() + 1)
-    file = f"/sys/block/nvme2c2n1/queue/io_timeout"
-    io_timoout = int(subprocess.run(["sudo", "cat", file], capture_output=True).stdout)
-    print(f"io_timeout: {io_timoout}")
-    assert io_timoout == 33000
+    file = f"/sys/block/{device_path}/queue/io_timeout"
+    io_timout = int(subprocess.run(["sudo", "cat", file], capture_output=True).stdout)
```
Comment (dsharma-dc) [inline tests/bdd/features/csi/node/test_parameters.py]: The fixed typo has a typo :)  - `io_timeout`

File: tests/bdd/features/csi/node/test_parameters.py [hunk 2]
```diff
@@ -54,19 +54,18 @@ def staging_a_volume(staging_a_volume):
...
+    print(f"io_timeout: {io_timout}")
+    assert io_timout == 33000
```
Comment (dsharma-dc) [inline tests/bdd/features/csi/node/test_parameters.py]: what's this existing magic number 33000 ? how's it derived.

**Review by dsharma-dc — APPROVED** [review]

**Review by pchandra19 — APPROVED** [review]

**Review by Abhinandan-Purkait — APPROVED** [review]

File: tests/bdd/features/csi/node/test_parameters.py (hunk 1 above)
Comment (tiagolobocastro) [reply tests/bdd/features/csi/node/test_parameters.py]: damn :D good catch

File: tests/bdd/features/csi/node/test_parameters.py (hunk 2 above)
Comment (tiagolobocastro) [reply tests/bdd/features/csi/node/test_parameters.py]: I added assert message, it's set by the csi-node on startup (33s), this asserts that it is 33300ms

**Comment by tiagolobocastro** [comment]
bors merge

**Pull request merged**